docs/guides/postgres/autoscaler/storage/cluster.md

```diff
@@ -0,0 +1,317 @@
+```
+
+#### Storage Autoscaling is set up successfully
+
+Let's check that the `postgresautoscaler` resource is created successfully,
+
+```bash
+$ kubectl get postgresautoscaler -n demo
+NAME           AGE
+pg-as-st   33s
+
+$ kubectl describe postgresautoscaler pg-as-st -n demo
+Name:         pg-as-st
+Namespace:    demo
+Labels:       <none>
+Annotations:  API Version:  autoscaling.kubedb.com/v1alpha1
+Kind:         PostgresAutoscaler
+Metadata:
+  Creation Timestamp:  2022-01-14T06:08:02Z
+  Generation:          1
+  Managed Fields:
+    ...
+  Resource Version:  24009
+  UID:               4f45a3b3-fc72-4d04-b52c-a770944311f6
+Spec:
+  Database Ref:
+    Name:  ha-postgres
+  Storage:
+    Mariadb:
+      Scaling Threshold:  20
+      Trigger:            On
+      Usage Threshold:    20
+Events:                   <none>
+```
+
+So, the `postgresautoscaler` resource is created successfully.
+
+Now, for this demo, we are going to manually fill up the persistent volume to exceed the `usageThreshold` using `dd` command to see if storage autoscaling is working or not.
+
+Let's exec into the database pod and fill the database volume(`/var/pv/data`) using the following commands:
+
+```bash
+$ kubectl exec -it -n demo ha-postgres-0 -- bash
+root@ha-postgres-0:/ df -h /var/pv/data
+Filesystem                                         Size  Used Avail Use% Mounted on
+/dev/topolvm/57cd4330-784f-42c1-bf8e-e743241df164 1014M  357M  658M  36% /var/pv/data
+root@ha-postgres-0:/ dd if=/dev/zero of=/var/pv/data/file.img bs=500M count=1
+1+0 records in
+1+0 records out
+524288000 bytes (524 MB, 500 MiB) copied, 0.340877 s, 1.5 GB/s
+root@ha-postgres-0:/ df -h /var/pv/data
+Filesystem                                         Size  Used Avail Use% Mounted on
+/dev/topolvm/57cd4330-784f-42c1-bf8e-e743241df164 1014M  857M  158M  85% /var/pv/data
+```
+
+So, from the above output we can see that the storage usage is 83%, which exceeded the `usageThreshold` 20%.
+
+Let's watch the `postgresopsrequest` in the demo namespace to see if any `postgresopsrequest` object is created. After some time you'll see that a `postgresopsrequest` of type `VolumeExpansion` will be created based on the `scalingThreshold`.
+
+```bash
+$ kubectl get postgresopsrequest -n demo
+NAME                         TYPE              STATUS        AGE
+pgops-ha-postgres-xojkua   VolumeExpansion   Progressing   15s
+```
+
+Let's wait for the ops request to become successful.
+
+```bash
+$ kubectl get postgresopsrequest -n demo
+NAME                         TYPE              STATUS       AGE
+pgops-ha-postgres-xojkua   VolumeExpansion   Successful   97s
+```
+
+We can see from the above output that the `PostgresOpsRequest` has succeeded. If we describe the `PostgresOpsRequest` we will get an overview of the steps that were followed to expand the volume of the database.
+
+```bash
+$ kubectl describe postgresopsrequest -n demo mops-ha-postgres-xojkua
```

Comment:
this command(kubectl describe postgresopsrequest -n demo mops-ha-postgres-xojkua) make it  (kubectl describe postgresopsrequest -n demo pgops-ha-postgres-xojkua )